magento/magento2
# Fix/issue 21564 2.3

### Description (*)
Admin order not enough quantity for item

### Fixed Issues (if relevant)
1. magento/magento2#21564: agento 2.2 Admin order not enough quantity for item

### Manual testing scenarios (*)
1. Login To magento backend
2. Go to Sales
3. Create new order
4. Select any customer.
5. select a product which have only one item remaining in stock (ex T-shirt)
6. a notification appears "We don't have as many "T-Shirt" as you requested. "
7. Press Update items and quantity, then this notification disappears

### Contribution checklist (*)
 - [*] Pull request has a meaningful description of its purpose
 - [*] All commits are accompanied by meaningful commit messages
 - [*] All new or changed code is covered with unit/integration tests (if applicable)
